### ISSUE TITLE
Titanic Monarch Zone Act One stuck on results screen.

### ISSUE THREAD

**Issue by SappYoin** [Mefiresu/RSDKv5-Decompilation#19]
So, after the boss, it seems to get stuck loading the next act.
All animations continue to play, and it looks like it's loading, but it never does.
I left it on for at least 15 minutes before shutting it off just in case it was loading times for whatever reason.
I have Mania Plus, though I didn't build the plus version of the port, because I assumed I wouldn't need it.

**Comment by Mefiresu** [comment]
This seems to be related to #18 somehow.
Does it keep happening or is it just a one time thing?
Also could you explain your current setup a bit more? (Wii/Dolphin, 240p mode or default, controller used...)

**Comment by SappYoin** [comment]
It keeps happening every time I beat the level.
I am using a wii, with 240p mode on, gamecube controller, and I'm using the 0.4.0 build.

**Comment by MegaBlue92** [comment]
Try deleting your settings.ini and run the game (let the game create a new settings.ini). Then try to beat the level and see if you progress. Im still at hydrocity because I haven't played recently.

**Comment by SappYoin** [comment]
huh, it let me through to the next act

**Comment by MegaBlue92** [comment]
Yep, it was same problem I had with Stardust Speedway Act 2 when I beat Metal Sonic.
